dash_labs/plugins/pages.py

```diff
@@ -9,22 +9,28 @@
 from os.path import isfile, join
 from textwrap import dedent
 from urllib.parse import parse_qs
+import re
 
-if not os.path.exists("pages"):
-    raise Exception("A folder called `pages` does not exist.")
 
 _ID_CONTENT = "_pages_plugin_content"
 _ID_LOCATION = "_pages_plugin_location"
-_ID_DUMMY = "_pages_plugin_dummy"
+_ID_STORE = "_pages_plugin_store"
+_ID_DUMMY = "pages_plugin_dummy"
```

Comment:
```suggestion
_ID_DUMMY = "_pages_plugin_dummy"
```
Was that change intentional?

Reply:
oops that was a typo